src/java.desktop/share/classes/sun/awt/util/IdentityArrayList.java

```diff
@@ -216,8 +216,12 @@ public boolean contains(Object o) {
      * Returns the index of the first occurrence of the specified element
      * in this list, or -1 if this list does not contain the element.
      * More formally, returns the lowest index {@code i} such that
-     * {@code Objects.equals(o, get(i))},
+     * {@code get(i)==o},
```

Comment:
Shall it have spaces on either side of `==`?
```suggestion
     * {@code get(i) == o},
```

I see `IdentityLinkedList.java` doesn't have spaces around `==` in this case. Yet in this case, probably `{@code o == e}` should not have the spaces too.